CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+### 3.13.2 - [#121](https://github.com/openfisca/country-template/pull/121)
+
+* Minor change
+* Details:
+  - Delete macOS related comment in bootstrap script
```

Comment:
There was a mix-up with the previous minor version I think 😅

Reply:
This description is not correct 😉